src/Resources/translations/SonataUserBundle.bg.xliff

```diff
@@ -42,6 +42,10 @@
         <source>users</source>
         <target>Потребители</target>
       </trans-unit>
```

Comment:
Shouldn't we remove the `users` translation ? which is now show.group_users

Reply:
That translation is used on the side menu. So both translations are needed.